### PR TITLE
Add AlertifyJS

### PR DESCRIPTION
**Pull request by MohammadYounes**
http://www.alertifyjs.com

### CODE REVIEW

File: files/alertifyjs/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "alertifyjs",
+  "repo": "mohammadyounes/alertifyjs",
+  "files": {
+    "basePath": "build/",
+    "exclude": ["alertify.zip"]
```
Comment (jimaek) [inline files/alertifyjs/update.json]: Please remove your exclude and use the following instead. (format it properly at first)
include: *.min.js, ./css/**/*.min.css

Comment (MohammadYounes) [inline files/alertifyjs/update.json]: Done!

**Comment by jimaek** [comment]
Great, can you either squash your commits or create a new PR? We have a max of 2 commits per PR

**Comment by MohammadYounes** [comment]
Oh! give me a minute till I clone it locally.

**Comment by MohammadYounes** [comment]
@jimaek  I'm on a slow connection, I created another PR #4458

**Pull request closed without merging**